Author: Helmut Grohne <helmut@subdivi.de>
Bug-Debian: https://bugs.debian.org/900019
Last-Update: 2018-05-24

--- pokerth-1.1.2.orig/connectivity.pro
+++ pokerth-1.1.2/connectivity.pro
@@ -41,13 +41,13 @@ unix : !mac {
 
 	QMAKE_LIBDIR += lib $${PREFIX}/lib /opt/gsasl/lib
 	INCLUDEPATH += $${PREFIX}/include
-	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system(qmake -query QT_INSTALL_LIBS)
+	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	BOOST_PROGRAM_OPTIONS = boost_program_options boost_program_options-mt
 	BOOST_SYS = boost_system boost_system-mt
 
 
 	#
-	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system(qmake -query QT_INSTALL_LIBS)
+	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	# to override the default '/usr' pass PREFIX
 	# variable to qmake.
 	#
--- pokerth-1.1.2.orig/load.pro
+++ pokerth-1.1.2/load.pro
@@ -42,14 +42,14 @@ unix : !mac {
 
 	QMAKE_LIBDIR += lib $${PREFIX}/lib /opt/gsasl/lib
 	INCLUDEPATH += $${PREFIX}/include
-	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system(qmake -query QT_INSTALL_LIBS)
+	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	BOOST_THREAD = boost_thread boost_thread-mt
 	BOOST_PROGRAM_OPTIONS = boost_program_options boost_program_options-mt
 	BOOST_SYS = boost_system boost_system-mt
 
 
 	#
-	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system(qmake -query QT_INSTALL_LIBS)
+	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	# to override the default '/usr' pass PREFIX
 	# variable to qmake.
 	#
--- pokerth-1.1.2.orig/pokerth_game.pro
+++ pokerth-1.1.2/pokerth_game.pro
@@ -414,7 +414,7 @@ unix:!mac {
 		LIB_DIRS = $${PREFIX}/lib \
 			$${PREFIX}/lib64 \
                         $${PREFIX}/lib/x86_64-linux-gnu \
-			$$system(qmake -query QT_INSTALL_LIBS)
+			$$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	}
 	android{
 		LIBPATH += $${PREFIX}/lib/armv7
@@ -435,7 +435,7 @@ unix:!mac {
 	BOOST_CHRONO = boost_chrono \
 		boost_chrono-mt
 
-	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system(qmake -query QT_INSTALL_LIBS)
+	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	# to override the default '/usr' pass PREFIX
 	# variable to qmake.
 	for(dir, LIB_DIRS):exists($$dir) {
--- pokerth-1.1.2.orig/pokerth_server.pro
+++ pokerth-1.1.2/pokerth_server.pro
@@ -180,7 +180,7 @@ unix : !mac {
 	INCLUDEPATH += $${PREFIX}/include
 	# see issue https://github.com/pokerth/pokerth/issues/282
 	INCLUDEPATH += $${PREFIX}/include/libircclient
-	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system(qmake -query QT_INSTALL_LIBS)
+	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	BOOST_FS = boost_filesystem boost_filesystem-mt
 	BOOST_THREAD = boost_thread boost_thread-mt
 	BOOST_PROGRAM_OPTIONS = boost_program_options boost_program_options-mt
@@ -191,7 +191,7 @@ unix : !mac {
 	BOOST_CHRONO = boost_chrono boost_chrono-mt
 
 	#
-	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system(qmake -query QT_INSTALL_LIBS)
+	# searching in $PREFIX/lib, $PREFIX/lib64 and $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	# to override the default '/usr' pass PREFIX
 	# variable to qmake.
 	#
--- pokerth-1.1.2.orig/zlib_compress.pro
+++ pokerth-1.1.2/zlib_compress.pro
@@ -48,7 +48,7 @@ unix : !mac {
 	#QMAKE_CXXFLAGS += -ffunction-sections -fdata-sections
 	#QMAKE_LFLAGS += -Wl,--gc-sections
 
-	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system(qmake -query QT_INSTALL_LIBS)
+	LIB_DIRS = $${PREFIX}/lib $${PREFIX}/lib64 $$system($$QMAKE_QMAKE -query QT_INSTALL_LIBS)
 	BOOST_FS = boost_filesystem boost_filesystem-mt
 	BOOST_IOSTREAMS = boost_iostreams boost_iostreams-mt
 	BOOST_SYSTEM = boost_system boost_system-mt
